napari/_qt/qt_main_window.py

```diff
@@ -157,6 +154,23 @@ def __init__(self, viewer, *, show: bool = True):
         if show:
             self.show()
 
+    def __getattr__(self, name):
+        if name == 'raw_stylesheet':
+            import warnings
+
+            warnings.warn(
+                (
+                    "The 'raw_stylesheet' attribute is deprecated and will be"
+                    "removed in version 0.4.7.  Please use "
```

Comment:
please. Left at least 2 versions between removal.

Reply:
we can do that.  but just out of curiosity what does that buy you?  You can access the exact same value from `qt.get_stylesheet()`... why do you want to be able to access it here across multiple deprecated versions?

Reply:
Personally, I do not use it. But I do not like the idea of 1 version deprecation.

Reply:
I'm 99% sure that you're the only person who would be using it! :joy:

Reply:
i'm then fine leaving this at a 1 version deprecation - this is inside out Qt folder and we're still in alpha.